source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -230,10 +230,13 @@ Example
 
 The following example performs these bulk operations:
 
-- An insert operation for a document where the ``_id`` is "3"
-- A replace operation for a document where the ``_id`` is "1" with a document that contains an additional field
-- An update operation for a document where the ``_id`` is "3" to a document that contains an additional field
-- A delete operation for all documents that contain the field ``x`` with the value "2"
+- An insert operation for a document where the ``name`` is ``"Zaynab Omar"`` and the
+  ``age`` is ``37``
+- A replace operation for a document where the ``_id`` is ``1`` with a new document
+  that contains the ``location`` field
+- An update operation for a document where the ``name`` is ``"Zaynab Omar"`` to change the ``name``
+  field to ``"Zaynab Hassan"``
+- A delete operation for all documents that have an ``age`` value greater than ``50``
```

Comment:
S: suggestion for clarity/syntax (didn't include it here, but also a good idea to specify which document changing instead of just "a document"--e.g., "updates the first matching document...")

```suggestion
- An operation that inserts a document where the ``name`` is ``"Zaynab Omar"`` and the
  ``age`` is ``37``
- An operation that replaces the document where the ``_id`` is ``1`` with a new document
  that contains the ``location`` field
- An operation that updates a document where the ``name`` is ``"Zaynab Omar"`` to change the ``name``
  field to ``"Zaynab Hassan"``
- An operation that deletes all documents where the ``age`` value is greater than ``50``
```